### DISCUSSION TITLE
let wrapped label of button align left?

### DISCUSSION THREAD

**Discussion by retsyo** [zauberzeug/nicegui#2034] [Q&A] [answered]
### Question

The following code wraps lable of button, we can find that the texts on 2nd lines is centered. But how to let all lines align left? Thanks

```python
from nicegui import Client, app, ui

with ui.column().classes('content-left'):
    strTmp = 'hello ' * 10
    ui.button(strTmp, icon="help") \
        .props('autofocus outlined rounded item-aligned="left" align="left" no-caps') \
        .classes('w-full')

ui.run()

``` 

![how2alignleft](https://github.com/zauberzeug/nicegui/assets/7960913/8f198afb-137b-406b-a5f4-7fc51a1d208a)

**Comment by falkoschindler** [comment, marked as answer]
This is a bit tricky, because Quasar automatically applies `text: center` to a nested span element.
The easiest workaround I could come up with is to wrap a label with custom text alignment:
```py
with ui.button().classes('w-16'):
    ui.label('Hi there!').classes('text-left')
```

**Reply by retsyo** [reply]
An off-topic question, what is the official name of such a `button`? A nested button, a mother button, a container button, or something else?

**Reply by falkoschindler** [reply]
Container button or button container sounds about right. But basically it's a normal button with a child element in its default slot.
